mlfinlab/labeling/fixed_time_horizon.py

```diff
@@ -0,0 +1,52 @@
+"""
+Chapter 3.2 Fixed-Time Horizon Method
+Described in "Classification-based Financial Markets Prediction using Deep Neural Networks" Dixon et al. (2016)
```

Comment:
Please add that Chapter 3.2 relates to the book **Advances in Financial Machine Learning**

Also please specify which algorithm in particular is described in the mentioned paper.